cmd/podman/sigproxy.go

```diff
@@ -25,6 +25,8 @@ func ProxySignals(ctr *libpod.Container) {
 
 			if err := ctr.Kill(uint(s.(syscall.Signal))); err != nil {
 				logrus.Errorf("Error forwarding signal %d to container %s: %v", s, ctr.ID(), err)
+				signal.StopCatch(sigBuffer)
```

Comment:
What does this do? Why are we pausing signal handling on error?

Reply:
Right now if you hit ^C when the system gets into this state you can not kill the container.